src/pint/models/tcb_conversion.py

```diff
@@ -19,7 +26,7 @@
 IFTE_K = 1 + IFTE_KM1
 
 
-def scale_parameter(model, param, n, backwards):
+def scale_parameter(model: TimingModel, param: str, n: int, backwards: bool):
```

Comment:
should this have `-> None` for the return?

Reply:
added